activity/kafkapub/ui/activity.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "tibco-kafkapub",
+  "version": "0.0.1",
+  "type": "flogo:activity",
+  "ref": "github.com/wnichols/flogo-contrib/activity/kafkapub",
```

Comment:
Should this point to github.com/TIBCOSoftware/flogo-contrib/activity/kafkapub instead?